src/Plugin/QueueWorker/AbstractPostProcessorQueueWorker.php

```diff
@@ -168,7 +167,7 @@ public function processItem($data) {
     $processor_instance = $this->getProcessorPlugin($data->plugin_config_entity_id);
 
     if (!$processor_instance) {
-      $this->logger->log(LogLevel::ERROR, 'Strawberry Runners Processing aborted because the @processor may be inactive', ['@processor' => $processor_instance->label()]);
+      $this->logger->log(LogLevel::ERROR, 'Strawberry Runners Processing aborted because the @processor queue processor may be inactive', ['@processor' => $data->plugin_config_entity_id]);
```

Comment:
@patdunlavey this is not really a queue processor but a Strawberry Runner Processor so will change this label during my pull (sorry)